server/src/main/java/io/druid/client/selector/ServerSelectorStrategy.java

```diff
@@ -32,5 +34,7 @@
 })
 public interface ServerSelectorStrategy
 {
-  public QueryableDruidServer pick(Set<QueryableDruidServer> servers, DataSegment segment);
+  QueryableDruidServer pick(Set<QueryableDruidServer> servers, DataSegment segment);
+
+  List<QueryableDruidServer> pick(Set<QueryableDruidServer> servers, DataSegment segment, int numServersToPick);
```

Comment:
anyone who has a custom `ServerSelectorStrategy` will not be forced to upgrade their code